snmp/datadog_checks/snmp/snmp.py

```diff
@@ -697,8 +697,8 @@ def submit_metric(self, name, snmp_value, forced_type, tags, options, extract_va
         except Exception as e:
             msg = (
                 'Unable to submit metric `{}` with '
-                'value=`{}` ({}), forced_type=`{}`, tags=`{}`, options=`{}`: {}'.format(
-                    name, snmp_value, type(snmp_value), forced_type, tags, options, e
+                'value=`{}` ({}), forced_type=`{}`, tags=`{}`, options=`{}`, extract_value_pattern=`{}`: {}'.format(
```

Comment:
is the '`' to print an apex ?

Reply:
Hum, what do you mean by "apex" ?

The '`' is just here to enclose values to make it more obvious where the value start/end.

Reply:
Yes meant the '`', yesterday in code I found out about the usage of `{!r}` in Python to keep outer quotes in strings, wondered what is the common way of including them.

Reply:
Not sure there is a standard way, I think it's fine as long as the log is clear to the readers.

`{!r}` (call __repr__) is probably fine too. It will depend more on how __repr__ is implemented by the object being logged.